flutter/shell/platform/tizen/tizen_window_ecore_wl2.cc

```diff
@@ -357,7 +357,8 @@ void TizenWindowEcoreWl2::RegisterEventHandlers() {
             if (!handled) {
               self->view_delegate_->OnKey(
                   key_event->key, key_event->string, key_event->compose,
-                  key_event->modifiers, key_event->keycode, true);
+                  key_event->modifiers, key_event->keycode,
+                  ecore_device_name_get(key_event->dev), true);
```

Comment:
question: Doesn’t input-event by the input panel update the device name?

Reply:
The resulting name is `ime` when using the input panel (software keyboard). The `ECORE_EVENT_KEY_DOWN` event is triggered only once (when the panel is being closed).